### PR TITLE
create page for the April event

### PR DESCRIPTION
**Pull request by slashmili**
It's easier to read it from https://github.com/elixir-berlin/planning/blob/2020-april-event/events/2020/april.md

### CODE REVIEW

File: events/2020/april.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# #69 April Talk and socialize
```
Comment (alexanderspace100) [inline events/2020/april.md:1]:
```suggestion
# #70 April Talk and socialize
```

File: events/2020/april.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+You may just watch the youtbue stream at [TBA] at 19:00
+
+### Q&A and socializing
+
+We are going to use Discord as way to get the audience engage with the speakers and with others.
+
+After joining the Discord server, you'll see the following voice channels:
+
+* **April Meetup**: Join this channel to ask the speakers your questions or if you want to share something with everyone
+* **Pantry**: If you've missed the socializing aspect of the meetup, join this channel and someone will be there to chit-chat with you!
```
Comment (alexanderspace100) [inline events/2020/april.md:24]: Sorry, what the difference between the Pantry and Water Cooler. Except for the name 🤔

File: events/2020/april.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* **Water Cooler** : If you've missed the socializing aspect of the meetup, join this channel and someone will be there to chit-chat with you!
+* **Web Corner** : to discuss web related stuff
```
Comment (alexanderspace100) [inline events/2020/april.md:26]: Same for those two

File: events/2020/april.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* **Embedded Corner** : to discuss web related stuff
+We are also 
+
+Instructions:
+1. Visit [https://discord.gg/WaKTXHD](https://discord.gg/WaKTXHD)
+2. Loging if you have an account in Discord, otherwise create an account(needs username, email and password)
```
Comment (alexanderspace100) [inline events/2020/april.md:32]: Maybe here we should specify that they should download the app. There is a significant difference in functionality compared with the browser (you know, I was the guinea pig) and it is not obvious for everyone that/why they should it.

File: events/2020/april.md (hunk 2 above)
Comment (slashmili) [reply events/2020/april.md:24]: Just to have two places so not everyone goes to the same place 😬

File: events/2020/april.md (hunk 3 above)
Comment (slashmili) [reply events/2020/april.md:26]: @Overbryd suggested to have multiple places for people to gather and chat

File: events/2020/april.md (hunk 1 above)
Comment (slashmili) [reply events/2020/april.md:1]: We'll we skipped the March event, which makes this one 70. We can also silently ignore that fact and use 70 here :D

Comment (alexanderspace100) [reply events/2020/april.md:1]: Didn't know we count like that. Then, is up to you :)

File: events/2020/april.md (hunk 2 above)
Comment (alexanderspace100) [reply events/2020/april.md:24]: Ahh, okay. Clever idea 👍

File: events/2020/april.md (hunk 3 above)
Comment (alexanderspace100) [reply events/2020/april.md:26]: Got it now!

**Comment by slashmili** [comment]
Applied the suggestions

**Comment by duksis** [comment]
👍 thanks!

**Pull request merged**